assets/dns/cluster-role.yaml

```diff
@@ -13,3 +13,13 @@ rules:
   verbs:
   - list
   - watch
+- apiGroups:
+  - "toleration.scheduling.openshift.io"
+  resources:
+  - ["*"]
```

Comment:
List of lists seems wrong; shouldn't that be the following?
```yaml
  resources:
  - "*"
```
or
```yaml
  resources: ["*"]
```

Reply:
ping @ravisantoshgudimetla